schema/va-spec/acmg-2015/pathogenicity-statement-profile-source.yaml

```diff
@@ -0,0 +1,32 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/acmg-2015/pathogenicity-statement-profile-source.yaml"
+title: ACMG 2015 Variant Pathogenicity Statement Standard Profile
+strict: true
+
+imports:
+  va-spec: ../base/pathogenicity-statement-profile-source.yaml
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  VariantPathogenicityStatement:
+    maturity: draft
+    description: >-
+      A Statement describing the role of a variant in causing an inherited condition.
+    allOf:
+    - $ref: "/ga4gh/schema/va-spec/1.x/base/VariantPathogenicityStatement"
+    - properties:
+        classification:
+          # this is a MappableConcept but we would ideally like to specify a value set with meta characteristics to validate with some consistency
```

Comment:
We should discuss this general concern when patterns like this occur. And they will occur fairly often during profiling.

this is a MappableConcept but we would ideally like to specify a value set with meta characteristics to validate with some consistency

Reply:
I'm not following; if we uncomment the code below, does this not do what we want? What are the meta characteristics you are describing here?

Reply:
moved to new PR